examples/cifar10/pt/learners/cifar10_learner.py

```diff
@@ -229,12 +215,10 @@ def save_model(self, is_best=False):
         else:
             torch.save(save_dict, self.local_model_file)
 
-    def _train(
-        self,
-        shareable: Shareable,
-        fl_ctx: FLContext,
-        abort_signal: Signal,
-    ) -> Shareable:
+    def train(self, shareable: Shareable, fl_ctx: FLContext, abort_signal: Signal) -> Shareable:
+        if not self.initialized:
```

Comment:
This shouldn't be needed, since the system guarantee the call of START_RUN event before any task.

Reply:
Thanks. Removed.